bin/check-release-environment

```diff
@@ -6,6 +6,10 @@ if [ -z "${STAINLESS_API_KEY}" ]; then
   errors+=("The STAINLESS_API_KEY secret has not been set. Please contact Stainless for an API key & set it in your organisation secrets on GitHub.")
 fi
 
+if [ -z "${PYPI_TOKEN}" ]; then
+  errors+=("The ANTHROPIC_PYPI_TOKEN secret has not been set. Please set it in either this repository's secrets or your organisation secrets.")
```

Comment:
@RobertCraigie typo here and above

Reply:
sorry, this is actually just not the american spelling - but the comment above still stands